.github/workflows/tests.yml

```diff
@@ -39,12 +39,12 @@ jobs:
         if: matrix.os != 'ubuntu-latest'
         env:
           BROWSER: ${{ matrix.browser }}
-        run: dotnet test ./src/Playwright.Tests/Playwright.Tests.csproj -c Debug
+        run: dotnet test ./src/Playwright.Tests/Playwright.Tests.csproj -c Debug -f net6.0
```

Comment:
before tests were running on netstandard2.0 and net6.0. We test it on a different pipeline, so lets revert this how it previously was.